.github/workflows/test-runner-on-perf.yml

```diff
@@ -109,13 +110,21 @@ jobs:
           echo "Apply helm updates, including updated data-repo version"
           helmfile apply
           cd ${GITHUB_WORKSPACE}/${workingDir}
-      - name: "Cycle pods to clear lingering artifacts if not doing version bump"
-        if: ${{ steps.read_property.outputs.LATEST_VERSION == steps.read_property.outputs.CURRENT_SEMVER }}
+      - name: Clear Perf Database
+        env:
+          ORG_GRADLE_PROJECT_dbDatarepoUri: jdbc:postgresql://localhost:5434/datarepo
+          ORG_GRADLE_PROJECT_dbDatarepoPassword: ${{ secrets.PERF_DB_DATAREPO_PWD }}
+          ORG_GRADLE_PROJECT_dbStairwayUri: jdbc:postgresql://localhost:5434/stairway
+          ORG_GRADLE_PROJECT_dbStairwayPassword: ${{ secrets.PERF_DB_DATAREPO_PWD }}
         run: |
-          echo "Cycle API pods to trigger migrations to run and clear out database"
-          kubectl delete pods -n perf -l app.kubernetes.io/name=datarepo-api
-          echo "Sleep 45 seconds to give the pods a chance to start cycling before checking if up and on correct version"
-          sleep 45
+          # port foward to perf database
+          kubectl port-forward --namespace perf svc/perf-jade-gcloud-sqlproxy 5434:5432 & \
+          sleep 10 && \
+          echo "After 10 seconds - run drop all & update" && \
+          cd ${GITHUB_WORKSPACE}/${workingDir}  && \
+          ./gradlew dropAll && \
+          ./gradlew update && \
+          exit 0
       - name: "Wait for Perf Cluster to come back up with correct version"
```

Comment:
I recognize that we just spent a sprint pulling things like this into the github-actions repo. 

What do we want our norms around putting things in github-actions? 
1. What if the code is only used in one place?
2. What if it's new code that we want to test out a bit before moving it to a slightly harder-to-edit location?